### PR TITLE
Disabling dark theme

### PR DESCRIPTION
**Pull request by evertonfraga**
In order to keep Mist interface always consistent and prevent visual glitches (#523, #702), we're dropping the dark theme, used on Linux.

### CODE REVIEW

**Comment by evertonfraga** [comment]
Also fixed some blank spaces above the sidebar and dapp-url bar. Properly tested on ubuntu 64. 

screen shot 2016-10-28 at 1 31 03 pm
screen shot 2016-10-28 at 1 31 21 pm
screen shot 2016-10-28 at 1 40 50 pm

**Comment by frozeman** [comment]
otherwise very nice, thanks for the fast PR!

File: interface/client/styles/layout.import.less [hunk 1]
```diff
@@ -118,16 +105,6 @@ main {
     left: @widthSideBar;
     top: 0; 
     bottom: 0;
-
-    &.open-bar {
```
Comment (evertonfraga) [inline interface/client/styles/layout.import.less]: back in time when you focused on the url bar, a huge div unfolded, and blurred the active tab. as we don't have this unfolding section anymore, i did the clearance.

Comment (frozeman) [inline interface/client/styles/layout.import.less]: ok super

Comment (evertonfraga) [inline interface/client/styles/layout.import.less]: @frozeman now in Mist repo at least 01 review is required in order to merge. if you approve the PR I can merge it right away.

**Pull request merged**